website/docs/notes/concepts.md

```diff
@@ -21,52 +19,71 @@ To achieve this, MMF has few opinions about architecture of your research projec
+| Scene Text VQA       | stvqa                | vqa           | [paper: Scene Text Visual Question Answering](https://arxiv.org/abs/1905.13648)                                   |
+| Text Caps            | text_caps            | captioning    | [website](https://textvqa.org/textcaps)                                                                           |
+| TextVQA              | textvqa              | vqa           | [website](https://textvqa.org/)                                                                                   |
+| VisualDialog         | visdial              | dialog        | [website](https://visualdialog.org/)                                                                              |
+| Visual Entailment    | visual_entailment    |               | [github](https://github.com/necla-ml/SNLI-VE)                                                                     |
+| Visual Genome        | visual_genome        | vqa           | [website](https://visualgenome.org/)                                                                              |
+| VizWiz               | vizwiz               | vqa           | [website](https://vizwiz.org/)                                                                                    |
+| VQA 2.0              | vqa2                 | vqa           | [website](https://visualqa.org/)                                                                                  |
+| VQA CP V2            | vqacp_v2             | vqa           | [paper: Don’t Just Assume; Look and Answer - Overcoming Priors for Visual Question Answering](https://arxiv.org/pdf/1712.00377.pdf)|
 
-Following table shows the inverse of the above table, datasets along with their tasks and keys:
-
-|              |               |            |              |
-| ------------ | ------------- | ---------- | ------------ |
-| Datasets     | Key           | Task       | Notes        |
-| VQA 2.0      | vqa2          | vqa        |              |
-| TextVQA      | textvqa       | vqa        |              |
-| VizWiz       | vizwiz        | vqa        |              |
-| VisualDialog | visdial       | dialog     | Coming soon! |
-| VisualGenome | visual_genome | vqa        |              |
-| CLEVR        | clevr         | vqa        |              |
-| MS COCO      | coco          | captioning |              |
 
 ## Models
 
 Reference implementations for state-of-the-art models have been included to act as a base for reproduction of research papers and starting point of new research. MMF has been used in past for following papers:
 
 - [Towards VQA Models That Can Read (LoRRA model)](https://arxiv.org/abs/1904.08920)
 - [VQA 2018 Challenge winner](https://arxiv.org/abs/1807.09956)
-- VizWiz 2018 Challenge winner
+- [VizWiz 2018 Challenge winner](https://vizwiz.org/wp-content/uploads/2019/06/workshop2018_slides_FAIR_A-STAR.pdf)
```

Comment:
Point to models page instead. Just mention that each dataset and model have unique keys, add keys in the respective models and datasets pages.